src/servers/trigger.ts

```diff
@@ -53,7 +53,7 @@ import { NotificationsService } from "@services/notifications"
 import { activateLndHealthCheck, lndStatusEvent } from "@services/lnd/health"
 import { recordExceptionInCurrentSpan, wrapAsyncToRunInSpan } from "@services/tracing"
 
-import healthzHandler from "./middlewares/healthz"
+import healthzHandler from "./rest/healthz"
```

Comment:
I don't really see `rest` being a useful subcategory. It's one of those functional vs semantic groupings. how about:
```
server
- auth
  - router.ts
- health.ts
```
Ie I don't see the need for the rest subfolder - especially since `health` doesn't even conform to rest.

Reply:
renamed to `authorization` and move back health into middleware even thought it's not a middleware per `express` definition.